library/kani_macros/src/sysroot/loop_contracts/mod.rs

```diff
@@ -240,9 +243,30 @@ fn transform_break_continue(block: &mut Block) {
     block.stmts.push(return_stmt);
 }
 
+fn convert_loop_to_while(loop_expr: ExprLoop) -> Expr {
+    // Create a while expression with `true` as the condition
+    let while_expr = ExprWhile {
+        attrs: loop_expr.attrs, // Preserve any attributes
+        label: loop_expr.label, // Preserve any labels
+        while_token: Default::default(),
+        cond: Box::new(parse_quote!(true)), // Set condition to `true`
```

Comment:
Is this any different in outcome from using `cond: Expr::bool_constant(true),` ?